mongo/mongo.go

```diff
@@ -129,33 +129,45 @@ func createImageUpdateQuery(ctx context.Context, id string, image *models.Image)
 
 	log.Event(ctx, "building update query for image resource", log.INFO, log.INFO, log.Data{"image_id": id, "image": image, "updates": updates})
 
-	if image.CollectionID != "" {
-		updates["next.collection_id"] = image.CollectionID
-	}
-
-	if image.State != "" {
-		updates["next.state"] = image.State
-	}
-
-	if image.Filename != "" {
-		updates["next.filename"] = image.Filename
-	}
+	updates["collection_id"] = image.CollectionID
```

Comment:
So you can't partially update image document? e.g. I just want to update the collection ID without changing any other fields?

Reply:
I just sent a message to James, I don't know what is the requirement about this, I thought that we wanted to overwrite the whole object but maybe you're right because the description is:
```
Update image endpoint for adding object S3 path (PUT)
```

Reply:
Also the fields in swagger spec do not refer to being mandatory. If they are then there should be sufficient validation to prevent writing empty values to these fields (this might be the case already, I haven't checked).

Reply:
James suggested that for the MVP we should override the whole object:
`
I think for now PUT should overwrite the entire object as you say. We may want validation for different cases in future but not for MVP.`

Reply:
Maybe we could create new PUT endpoints for different fields int he future? (e.g. `PUT /images/{id}/type`)

Reply:
Possibly but for now if mvp is to overwrite whole object then Im happy with this